bin/clover/src/pipeline-steps/attachDefaultImportAndDiscoverFuncs.ts

```diff
@@ -1,34 +1,32 @@
 import _ from "lodash";
 import {
-  createDefaultManagementFuncs,
+  createDefaultImportAndDiscoverFuncs,
   createManagementFuncSpec,
 } from "../spec/funcs.ts";
 import { ExpandedPkgSpec } from "../spec/pkgs.ts";
 
-export function generateDefaultManagementFuncs(
+export function attachDefaultImportAndDiscoverFuncs(
   specs: ExpandedPkgSpec[],
 ): ExpandedPkgSpec[] {
-  const newSpecs = [] as ExpandedPkgSpec[];
-  const defaultMgmtFuncs = createDefaultManagementFuncs();
+  const defaultMgmtFuncs = createDefaultImportAndDiscoverFuncs();
 
   for (const spec of specs) {
-    const [schema] = spec.schemas;
-    const [schemaVariant] = schema.variants;
-    const funcs = spec.funcs;
-    const mgmtFuncs = schemaVariant.managementFuncs;
+    const { funcs, schemas: [{ variants: [variant] }] } = spec;
+    const { cfSchema } = variant;
+
+    // Don't generate import or discover for management functions
+    if (!cfSchema.handlers?.list) continue;
```

Comment:
I'm not sure this is quite right. If we don't have `list`, we can't do discover, but we could still do a single import as that requires `read` only. If we have `list`, but no `read`, we can't do discovery as it is written today as we do one then the other for each listed item. 

Discover = `list` + `read`
Import = `read`

Reply:
I'll check it, that sounds reasonable.